app/src/main/java/org/simple/clinic/registration/phone/PhoneNumberValidator.kt

```diff
@@ -0,0 +1,13 @@
+package org.simple.clinic.registration.phone
+
+import javax.inject.Inject
+
+class PhoneNumberValidator @Inject constructor() {
+
+  val LENGTH_OF_INDIAN_PHONE_NUMBER = 10
```

Comment:
The existence of this value lies only inside the scope of this class. What's the reason for moving it ouside?

Reply:
Because it currently is a class property, yet we are using it as a constant. If the intention is to use it as a a constant, then it should actually be a constant.

Reply:
Fixed